charts/tractusx-connector-azure-vault/templates/deployment-controlplane.yaml

```
@@ -0,0 +1,362 @@
+      {{- end }}
+      serviceAccountName: {{ include "txdc.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.controlplane.podSecurityContext | nindent 8 }}
+      initContainers:
+        {{- toYaml .Values.controlplane.initContainers | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.controlplane.securityContext | nindent 12 }}
+
+          # either use the specified image, or use the default one
+          {{- if .Values.controlplane.image.repository }}
+          image: "{{ .Values.controlplane.image.repository }}:{{ .Values.controlplane.image.tag | default .Chart.AppVersion }}"
+          {{- else }}
+          image: "tractusx/edc-controlplane-postgresql-azure-vault:{{ .Values.controlplane.image.tag | default .Chart.AppVersion }}"
+          {{- end }}
+          imagePullPolicy: {{ .Values.controlplane.image.pullPolicy }}
+          ports:
+          {{- range $key,$value := .Values.controlplane.endpoints }}
+            - name: {{ $key }}
+              containerPort: {{ $value.port }}
+              protocol: TCP
+          {{- end }}
+          {{- if .Values.controlplane.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: {{ .Values.controlplane.endpoints.observability.path }}/check/liveness
+              port: {{ .Values.controlplane.endpoints.observability.port }}
+            initialDelaySeconds: {{ .Values.controlplane.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.controlplane.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.controlplane.livenessProbe.timeoutSeconds }}
+            failureThreshold: {{ .Values.controlplane.livenessProbe.failureThreshold }}
+            successThreshold: {{ .Values.controlplane.livenessProbe.successThreshold }}
+          {{- end }}
+          {{- if .Values.controlplane.readinessProbe.enabled }}
+          readinessProbe:
+            httpGet:
+              path: {{ .Values.controlplane.endpoints.observability.path }}/check/readiness
+              port: {{ .Values.controlplane.endpoints.observability.port }}
+            initialDelaySeconds: {{ .Values.controlplane.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.controlplane.readinessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.controlplane.readinessProbe.timeoutSeconds }}
+            failureThreshold: {{ .Values.controlplane.readinessProbe.failureThreshold }}
+            successThreshold: {{ .Values.controlplane.readinessProbe.successThreshold }}
+          {{- end }}
+          resources:
+            {{- toYaml .Values.controlplane.resources | nindent 12 }}
+          env:
+          {{- if .Values.controlplane.debug.enabled }}
+            - name: "JAVA_TOOL_OPTIONS"
+            {{- if and .Values.controlplane.debug.enabled .Values.controlplane.debug.suspendOnStart }}
```

Does this change (and the same for the dataplane) need to be applied to the other charts as well?

```suggestion
            {{- if .Values.controlplane.debug.suspendOnStart }}
```